tests/functional/conflict_test.py

```diff
@@ -9,6 +11,12 @@
 from testing.python_lib import PYTHON_LIB
 
 
+def assert_venv_age(tmpdir):
```

Comment:
def assert_venv_marked_invalid(venv):
    """we mark a virtualenv as invalid by bumping its timestamp back by a day"""